packages/backend/src/controllers/membership-plan.ts

```diff
@@ -1,11 +1,17 @@
 import { RESTController } from '../libs/classes/RESTController';
 import { getRepository } from 'typeorm';
-import { MemberCtrl } from './member';
-import { Member } from '../models/Member';
-import { MembershipPlan } from '../models/MembershipPlan';
+import { Request, Response } from 'express';
+import { MembershipPlan, PlanType } from '../models/MembershipPlan';
 
 export class MembershipPlanCtrl extends RESTController<MembershipPlan> {
   constructor() {
     super(getRepository(MembershipPlan));
   }
+
+  public getAllTypes = async (
+    req: Request,
+    res: Response
+  ): Promise<Response> => {
+    return res.status(201).send(Object.values(PlanType).filter(type => typeof type === 'string'));
```

Comment:
Clearly, I think you can do better than filter with a typeof ==='string'. At least explain why you're doing like that.